marklogic-data-hub/src/main/resources/ml-modules/root/com.marklogic.hub/lib/entity-services-wrapper.xqy

```diff
@@ -19,6 +19,19 @@ module namespace es-wrapper = "http://marklogic.com/data-hub/es-wrapper";
 
 declare option xdmp:mapping "false";
 
+declare function es-wrapper:search-options-generate($model)
+{
+  xdmp:eval('
```

Comment:
why is this an eval instead of just calling es:search-options-generate()?

Reply:
this is part of the support for ML8. If this module didn't have the eval then the REST API would barf on the static check on ML8 when the code is deployed. If we drop ML8 support then this can go away.